### ISSUE TITLE
Please add support for Protobuf 5

### ISSUE THREAD

**Issue by filbranden** [tink-crypto/tink-py#40]
**Help us help you**

Planning a new virtual environment that includes Protobuf 5, more specifically `protobuf==5.28.1`, I'd like to request that a version of Tink that supports Protobuf 5 would be available

**Is your feature request related to a problem?**

Not a blocker right now, but planning to migrate applications to the new venv that includes Protobuf 5 and I expect the ones that use Tink will be blocked on this feature

**What sort of feature would you like to see?**

If support for Protobuf 5 is not backwards compatible, then possibly consider splitting into a new major version e.g. `tink==2.0.0` with support for Protobuf 5, keeping a Tink 1.x branch for Protobuf <5 in case it still needs maintenance?

**Have you considered any alternative solutions?**

Not really, we have at least one app that went all in on Tink so we'd really like to have Tink with Protobuf 5 support available.

**Would you like to add additional context?**

Using Python 3.12

Thanks!

**Comment by top-oai** [comment]
Hi there, just wanted echo this desire...  Is it just a matter of dropping the upper bound constraint?

**Comment by calderonth** [comment]
Yes please, this is quite problematic for us too when trying to use Tink in projects that rely on protobufs in 5.x, downgrading is not a great option.

**Comment by mouchar** [comment]
I noticed commit https://github.com/tink-crypto/tink-py/commit/f86f602679bca1a67315b5ff62e7bf9d2a984f16 solving this issue.
@juergw or @morambro - do you plan to cut a new release soon and publish wheels to PyPi?

**Comment by morambro** [comment]
Hi @mouchar, we are currently working on a new release. We don't have a precise timeline yet, but it should be ready by the end of March.

**Comment by juergw** [comment]
Tink Python 1.11 is now released, and Protobuf 5 should not work.
